frontend/src/routes/manage/Video/TechnicalDetails.tsx

```diff
@@ -115,35 +117,56 @@ export const TrackInfo: React.FC<TrackInfoProps> = (
         tracks.push({ resolution, mimetype, uri });
     }
 
+    const isSingleStream = flavors.size === 1;
+
     return <section css={className}>
         <h2>{t("manage.my-videos.technical-details.tracks")}</h2>
         <ul css={{ fontSize: 15, marginTop: 8, paddingLeft: 24 }}>
             {Array.from(flavors, ([flavor, tracks]) => <li key={flavor}>
-                {translateFlavors ? flavorTranslation(flavor) : <code>{flavor}</code>}
-                <ul>{tracks
+                {!isSingleStream && (translateFlavors
+                    ? flavorTranslation(flavor)
+                    : <code>{flavor}</code>)
+                }
+                <Tracks {...{ isSingleStream }}>{tracks
                     .sort((a, b) => (a.resolution?.[0] ?? 0) - (b.resolution?.[0] ?? 0))
                     .map((track, i) => <TrackItem key={i} {...track} />)
-                }</ul>
+                }</Tracks>
             </li>)}
         </ul>
     </section>;
 };
 
+type TracksProps = PropsWithChildren<{ isSingleStream: boolean }>
+const Tracks: React.FC<TracksProps> = ({ isSingleStream, children }) =>
+    isSingleStream ? <>{children}</> : <ul>{children}</ul>;
+
 const TrackItem: React.FC<SingleTrackInfo> = ({ mimetype, resolution, uri }) => {
     const { t } = useTranslation();
+    const type = mimetype && mimetype.split("/")[0] as "audio" | "video";
+    const subtype = mimetype && mimetype.split("/")[1];
```

Comment:
Also, maybe this might be useful:

```
const [type, subtype] = mimetype.split("/");
```

Reply:
I don't think this works, since we (or the compiler) doesn't know for sure whether a) mimetype exists and b) if it does, can be split like this.

Reply:
Oh wupps yeah, didn't think before commenting